src/game_api/script/usertypes/level_lua.cpp

```diff
@@ -1246,7 +1246,7 @@ void register_usertypes(sol::state& lua)
     auto customtheme_type = lua.new_usertype<CustomTheme>("CustomTheme", sol::constructors<CustomTheme(), CustomTheme(uint8_t, uint8_t), CustomTheme(uint8_t, uint8_t, bool)>(), sol::base_classes, sol::bases<ThemeInfo>());
     customtheme_type["level_file"] = &CustomTheme::level_file;
     customtheme_type["theme"] = &CustomTheme::theme;
-    // NoDoc
+    /// NoDoc
```

Comment:
is the NoDoc supposed to work here due to the autodoc changes? or doesn't matter now?

Reply:
It was working before since regex was just checking if there is `NoDoc` before the declaration
Now i changed it to get any comment that starts with `///` and then check if the comment is `NoDoc` or an actual comment